### ISSUE TITLE
Colab example fails with mixture

### ISSUE THREAD

**Issue by 8enmann** [google-research/text-to-text-transfer-transformer#64]
https://colab.research.google.com/drive/13shlV1wk9e9vcnpI77YvFbBBWAQKYACS

```py
task = t5.data.MixtureRegistry.get("glue_v002_proportional")
ds = task.get_dataset(split="validation", sequence_length={"inputs": 128, "targets": 32})
tfds.as_numpy(ds.take(5))
```

```py
/usr/local/lib/python3.6/dist-packages/t5/data/utils.py in assert_cached(self)
    608     """Raises an assertion error if cached dataset does not exist."""
    609     assert self.cached, (
--> 610         "'%s' does not exist in any of the task cache directories" % self.name)
    611 
    612   def get_cached_stats(self, split=tfds.Split.TRAIN):

AssertionError: 'glue_cola_v002' does not exist in any of the task cache directories
  In call to configurable 'rate_num_examples' (<function rate_num_examples at 0x7f025a3afae8>)
```

**Comment by craffel** [comment]
The glue_v002_proportional mixture uses rate_num_examples which requires cached tasks. You can create a hard-coded version of this mixture with the same (training) tasks and mixing proportions via

```
tasks_and_weights = [
  ('glue_cola_v002', 8551.0),
  ('glue_sst2_v002', 67349.0),
  ('glue_mrpc_v002', 3668.0),
  ('glue_qqp_v002', 363849.0),
  ('glue_stsb_v002', 5749.0),
  ('glue_mnli_v002', 392702.0),
  ('glue_qnli_v002', 104743.0),
  ('glue_rte_v002', 2490.0)
]
MixtureRegistry.add("glue_hardcoded", tasks_and_weights)
```

**Comment by adarob** [comment]
Reopening as we should consider making this the default in the external codebase.

**Comment by jinoobaek-qz** [comment]
https://github.com/google-research/text-to-text-transfer-transformer/pull/268

**Comment by adarob** [comment]
Fixed by #268
